doc/Changelog.md

```diff
@@ -3,6 +3,15 @@ Changelog {#changelog}
 
 # Release 1.2 (git master)
 
+* [90](https://github.com/BlueBrain/Tide/pull/90):
+  The fullscreen mode has some new features:
+  - Users can now enlarge and pan contents (without the need to use the zoom)
```

Comment:
does sound more confusing than actually solving a problem. Maybe I need to see it first or a another description which is more user-oriented :)